.travis.yml

```diff
@@ -7,6 +7,15 @@ install:
   - mvn -N io.takari:maven:wrapper -Dmaven=3.6.0
   - java -version && javac -version && ./mvnw -version -B
 
+before_script:
+  # Check each method has docs
+  - find vars -type f -name "*.groovy" -exec bash -c '[ -f vars/$(basename {} .groovy).txt ] || echo "Missing doc: - vars/$(basename {} .groovy).txt"' \;
+  - find vars -type f -name "*.groovy" -exec bash -c '[ -f vars/$(basename {} .groovy).txt ] || exit 1' \;
```

Comment:
If I'm reading this right we should have `$?` be `1` if we're missing docs but I'm not seeing that. Are you able to confirm?
```
$ find vars -type f -name "*.groovy" -exec bash -c '[ -f vars/$(basename {} .groovy).txt ] || echo "Missing doc: - vars/$(basename {} .groovy).txt"' \;
$ find vars -type f -name "*.groovy" -exec bash -c '[ -f vars/$(basename {} .groovy).txt ] || exit 1' \;
$ echo $?
0
$ touch vars/test.groovy
$ find vars -type f -name "*.groovy" -exec bash -c '[ -f vars/$(basename {} .groovy).txt ] || echo "Missing doc: - vars/$(basename {} .groovy).txt"' \;
Missing doc: - vars/test.txt
$ find vars -type f -name "*.groovy" -exec bash -c '[ -f vars/$(basename {} .groovy).txt ] || exit 1' \;
$ echo $?                                                                                               
0
```

Reply:
Simplified/corrected check